src/java/org/apache/cassandra/auth/AuthCache.java

```diff
@@ -225,6 +235,7 @@ public V get(K k)
         if (cache == null)
             return loadFunction.apply(k);
 
+        metrics.requests.mark();
```

Comment:
Aren't we missing the `getAll` path for tracking requests here? The `ChunkCache` itself encapsulates this logic in the `StatsCounter`.  It's worth adding such a test or a verification the metrics value for existing tests in `AuthCacheTest`.

Reply:
Hmm ... good catch.